CHANGELOG.md

```diff
@@ -1,6 +1,1484 @@
-# Backpack Android Changelog
```

Comment:
hello new unified changelog (again) - this will then also be aligned with using release-drafter or similar where it's just one set of release notes